src/main/java/com/conveyal/datatools/common/utils/NonRuntimeAWSException.java

```diff
@@ -0,0 +1,21 @@
+package com.conveyal.datatools.common.utils;
+
+import com.amazonaws.services.ec2.model.AmazonEC2Exception;
+
+/**
+ * A helper exception class that does not extend the RunTimeException class in order to make the compiler properly
+ * detect possible places where an exception could occur.
+ */
+public class NonRuntimeAWSException extends Exception {
```

Comment:
On the name of this class, should it be called something like `CheckedAWSException`? I actually like `NonRuntimeAWSException` as a name, but just figured I would play devil's advocate.

Reply:
I like CheckedAWSException better.